pydantic/_internal/_core_utils.py

```diff
@@ -48,6 +48,10 @@
 schema building because one of it's members refers to a definition that was not yet defined when the union
 was first encountered.
 """
+TAGGED_UNION_TAG_KEY = 'pydantic-tagged-union-tag'
```

Comment:
Let's change this value to match format of the others surrounding it:
```suggestion
TAGGED_UNION_TAG_KEY = 'pydantic.internal.tag'
```